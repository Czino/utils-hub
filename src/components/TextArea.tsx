import React from 'react'

export const TextArea = ({ className, ...props }: React.ComponentProps<'textarea'>) => (
  <textarea className={`px-2 py-2 w-full rounded border h-[300px] border-dark ${className}`} {...props}></textarea>
)
