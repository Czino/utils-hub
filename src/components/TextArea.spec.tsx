import { render } from '@testing-library/react'
import { TextArea } from './TextArea'

describe('TextArea', () => {
  it('should render TextArea with styles', () => {
    const { asFragment } = render(
      <TextArea className="mt-4" value="text">
        Test
      </TextArea>,
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
