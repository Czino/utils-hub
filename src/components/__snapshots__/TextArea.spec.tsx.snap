// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea should render TextArea with styles 1`] = `
<DocumentFragment>
  <textarea
    class="px-2 py-2 w-full rounded border h-[300px] border-dark mt-4"
  >
    text
  </textarea>
</DocumentFragment>
`;
